src/nwsapi.js

```diff
@@ -57,7 +57,7 @@
     HasEscapes: RegExp('\\\\'),
     HexNumbers: RegExp('^[0-9a-fA-F]'),
     EscOrQuote: RegExp('^\\\\|[\\x22\\x27]'),
-    RegExpChar: RegExp('(?:(?!\\\\)[\\\\^$.*+?()[\\]{}|\\/])', 'g'),
+    RegExpChar: RegExp('(?:(?!\\\\)[\\\\^$.,*+?()[\\]{}|\\/])', 'g'),
```

Comment:
not sure if escaping the comma will help

Reply:
@slavaleleka thank you so much for your help.
I believe this might be fixing the problem since in previous "nwmatcher" this was resolved.
The problem is that in new "nwsapi" I did change the way I split the comma "group" selector by avoiding attribute values, thus as you did find out in the new method we need to add the comma character in the RegExp list of charaters needed to be escaped.

After applying I will need to repeat all the testing to prepare for a new 2.2.3 release including your other fixes.

Again, thank you for the fixes and efforts you out in this.

Reply:
@slavaleleka
the problem was in the REX.SplitGroup which I did refactor.
It is the one responsible for the comma group parsing requiring a very specific
check about the comma and the square bracket not being part of an attribute value.

Closing this now.
Thank you for your help.